[2022-04-15 13:46:35,148] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_2 scheduled__2020-01-03T00:00:00+00:00 [queued]>
[2022-04-15 13:46:36,262] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_2 scheduled__2020-01-03T00:00:00+00:00 [queued]>
[2022-04-15 13:46:36,316] {taskinstance.py:1249} INFO - 
--------------------------------------------------------------------------------
[2022-04-15 13:46:36,317] {taskinstance.py:1250} INFO - Starting attempt 1 of 1
[2022-04-15 13:46:36,317] {taskinstance.py:1251} INFO - 
--------------------------------------------------------------------------------
[2022-04-15 13:46:37,273] {taskinstance.py:1270} INFO - Executing <Task(PythonOperator): task_2> on 2020-01-03 00:00:00+00:00
[2022-04-15 13:46:37,423] {standard_task_runner.py:52} INFO - Started process 101621 to run task
[2022-04-15 13:46:37,518] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'parallel_dag', 'task_2', 'scheduled__2020-01-03T00:00:00+00:00', '--job-id', '42', '--raw', '--subdir', 'DAGS_FOLDER/parallel_dag.py', '--cfg-path', '/tmp/tmptm4ijpal', '--error-file', '/tmp/tmpof9iloc9']
[2022-04-15 13:46:38,175] {standard_task_runner.py:80} INFO - Job 42: Subtask task_2
[2022-04-15 13:46:45,238] {logging_mixin.py:109} INFO - Running <TaskInstance: parallel_dag.task_2 scheduled__2020-01-03T00:00:00+00:00 [running]> on host 10.0.2.15
[2022-04-15 13:46:49,619] {taskinstance.py:1446} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=solfa
AIRFLOW_CTX_DAG_ID=parallel_dag
AIRFLOW_CTX_TASK_ID=task_2
AIRFLOW_CTX_EXECUTION_DATE=2020-01-03T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-01-03T00:00:00+00:00
[2022-04-15 13:46:52,696] {python.py:175} INFO - Done. Returned value was: None
[2022-04-15 13:46:54,721] {taskinstance.py:1278} INFO - Marking task as SUCCESS. dag_id=parallel_dag, task_id=task_2, execution_date=20200103T000000, start_date=20220415T134635, end_date=20220415T134654
[2022-04-15 13:46:55,729] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-04-15 13:46:58,028] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-04-15 15:54:51,409] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_2 scheduled__2020-01-03T00:00:00+00:00 [queued]>
[2022-04-15 15:54:51,455] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_2 scheduled__2020-01-03T00:00:00+00:00 [queued]>
[2022-04-15 15:54:51,456] {taskinstance.py:1249} INFO - 
--------------------------------------------------------------------------------
[2022-04-15 15:54:51,457] {taskinstance.py:1250} INFO - Starting attempt 1 of 1
[2022-04-15 15:54:51,457] {taskinstance.py:1251} INFO - 
--------------------------------------------------------------------------------
[2022-04-15 15:54:51,524] {taskinstance.py:1270} INFO - Executing <Task(PythonOperator): task_2> on 2020-01-03 00:00:00+00:00
[2022-04-15 15:54:51,528] {standard_task_runner.py:52} INFO - Started process 177644 to run task
[2022-04-15 15:54:51,542] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'parallel_dag', 'task_2', 'scheduled__2020-01-03T00:00:00+00:00', '--job-id', '138', '--raw', '--subdir', 'DAGS_FOLDER/parallel_dag.py', '--cfg-path', '/tmp/tmpna2ll9k6', '--error-file', '/tmp/tmpz8yjyza4']
[2022-04-15 15:54:51,546] {standard_task_runner.py:80} INFO - Job 138: Subtask task_2
[2022-04-15 15:54:51,646] {logging_mixin.py:109} INFO - Running <TaskInstance: parallel_dag.task_2 scheduled__2020-01-03T00:00:00+00:00 [running]> on host 10.0.2.15
[2022-04-15 15:54:51,724] {taskinstance.py:1446} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=solfa
AIRFLOW_CTX_DAG_ID=parallel_dag
AIRFLOW_CTX_TASK_ID=task_2
AIRFLOW_CTX_EXECUTION_DATE=2020-01-03T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-01-03T00:00:00+00:00
[2022-04-15 15:54:54,725] {python.py:175} INFO - Done. Returned value was: None
[2022-04-15 15:54:54,741] {taskinstance.py:1278} INFO - Marking task as SUCCESS. dag_id=parallel_dag, task_id=task_2, execution_date=20200103T000000, start_date=20220415T155451, end_date=20220415T155454
[2022-04-15 15:54:54,767] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-04-15 15:54:54,899] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
